css/styles.scss

```diff
@@ -19,16 +19,29 @@
 @import '../sass/components/_icons';
 @import '../sass/components/_skip-link';
 @import '../sass/components/_tables';
-@import '../sass/components/cd-footer/_cd-footer-copyright';
-@import '../sass/components/cd-footer/_cd-footer-menu';
-@import '../sass/components/cd-footer/_cd-footer-provided-by';
-@import '../sass/components/cd-footer/_cd-footer-social';
-@import '../sass/components/cd-footer/_cd-footer';
-@import '../sass/components/cd-header/_cd-dropdowns';
-@import '../sass/components/cd-header/_cd-header';
-@import '../sass/components/cd-header/_cd-logo';
-@import '../sass/components/cd-header/_cd-nav';
-@import '../sass/components/cd-header/_cd-search';
-@import '../sass/components/cd-header/_cd-sites-menu';
-@import '../sass/components/cd-header/_cd-user-menu';
-@import '../sass/components/cd-layout/_cd-layout';
+
+@import
```

Comment:
Bulk add all CD partials. See what might conflict with existing partials

Reply:
Commenting out partials that conflict. I will delete them eventually

Reply:
@rupl This file is likely what is causing any lack of styling of content.
I've enabled the skip-link import, but it uses a variable defined in _font-size partial so needed to enable that too.
I've commented out the rest because they were negatively affecting some styles, and they have equivalent (not necessarily duplicate) rules from the styleguide that are more up-to-date (anything prefixed with cd-)

I'm still trying to unravel it all.